src/api/app/assets/stylesheets/webui/layout.scss

```diff
@@ -112,17 +112,17 @@
 
 #footer, #footer-legal {
   & a {
-    color: $gray-700;
-    &:hover { color: $gray-900; }
+    color: var(--bs-secondary-color);
+    &:hover { color: var(--bs-body-color); }
   }
 }
```

Comment:
I have just realized, that we redefined here the styles of the `#footer `element defined in the `sticky-footer.scss` file: https://github.com/openSUSE/open-build-service/blob/9d0b803b6dfbdd80811c92b503c649db6e48a342/src/api/app/assets/stylesheets/webui/sticky-footer.scss#L9-L32

 Could be this PR followed by another PR refactoring those classes, at least?